_posts/2018-06-01-debian.md

```diff
@@ -5,12 +5,12 @@ title: "Debian"
 date: "2018-06-01"
```

Comment:
The date should also be modified, it's "Page last updated" on <https://computefreely.org/distro/debian>.